environments/drupal-8.sh

```diff
@@ -29,7 +28,8 @@ function drupal_ti_ensure_module_linked() {
 	fi
 
 	composer config repositories.$DRUPAL_TI_MODULE_NAME path $TRAVIS_BUILD_DIR
-	composer require drupal/$DRUPAL_TI_MODULE_NAME
+	composer config repositories.drupal composer https://packages.drupal.org/8
```

Comment:
Apparently it works in a "first come first serve" fashion, rather than a cascading override.